examples/2016-10-31/iot_backend/index.js

```diff
@@ -0,0 +1,30 @@
+console.log('Loading function');
+
+var AWS = require('aws-sdk');
+var dynamo = new AWS.DynamoDB.DocumentClient();
+var table = process.env.TABLE_NAME;
+
+exports.handler = function(event, context) {
```

Comment:
You should use 3 argument syntax: function(event, context, callback) and call callback(err) instead of context.fail() and callback(null) instead of context.done(). This is the recommended approach